awx/api/views/__init__.py

```diff
@@ -2805,8 +2817,9 @@ def post(self, request, *args, **kwargs):
         limit = host.name
 
         # NOTE: We limit this to one job waiting per host per callblack to keep them from stacking crazily
-        if Job.objects.filter(status__in=['pending', 'waiting', 'running'], job_template=job_template,
-                              limit=limit).count() > 0:
+        if models.Job.objects.filter(
+            status__in=['pending', 'waiting', 'running'], job_template=job_template, limit=limit
+        ).count() > 0:
```

Comment:
I know this isn't directly relevant for this cleanup work, but is there any reason that this shouldn't be a `.exists()` instead?

Reply:
No, an `.exists()` would probably work here, too.